contracts/interchain-token-service/src/contract.rs

```diff
@@ -534,11 +532,8 @@ impl InterchainTokenService {
                     ContractError::InvalidTokenMetaData
                 );
 
-                // Note: attempt to convert a byte string which doesn't represent a valid Soroban address fails at the Host level
-                let minter = minter
-                    .map(|m| Address::from_xdr(env, &m))
-                    .transpose()
-                    .map_err(|_| ContractError::InvalidMinter)?;
+                // Note: convert from Option<Bytes> to Option<Address> type
```

Comment:
```suggestion
                // Note: converting a byte string which doesn't represent a valid Soroban address fails at the Host level
```
the prior note is still valid since there's no explicit Result type being returned on a failure from this method